contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -4503,6 +4514,45 @@ pltsql_validator(PG_FUNCTION_ARGS)
 		}
 
 		ReleaseSysCache(tuple);
+		
+		/* 
+		 * If the function has TVP in its arguments or function body 
+		 * it should be declared as VOLATILE by default 
+		 */
+		if(prokind == PROKIND_FUNCTION && (has_table_var && !is_itvf && !is_mstvf))
```

Comment:
Why arent we changing it for mstvf and istvf??

Reply:
in `gram-tsql-rule.y` there is separate rules for Inline Table Valued Function(itvf), Multi Statement Table Valued Function(mstvf) and Normal create Function.
So, in `gram-tsql-rule.y` I have added an option of `STABLE` only for normal create function rule. 
So for itvf and mstvf no volatility option is provided, hence it will be `VOLATILE` by default